.github/workflows/autotest.yml

```diff
@@ -34,6 +47,7 @@ jobs:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         goos: ${{ matrix.goos }}
         goarch: ${{ matrix.goarch }}
+        goamd64: ${{ matrix.goamd64 }}
```

Comment:
Maybe it wll fail due to `go1.16`?

Reply:
In previous versions this variable would have been an empty string and should not affect the build.